test/controllers/api/v1/items/business_controller_test.rb

```diff
@@ -0,0 +1,59 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.